### ISSUE TITLE
mobilenet-ssd  accuracy is very low for windows-version (voc0712)

### ISSUE THREAD

**Issue by liyuming1978** [01org/caffe#102]
caffe version:  commit 2e7138570d324564c80c040b83ef6b1d4b39324d
(Merge pull request #100 from listenlink/fix_python_binding  )

how to build windows caffe:
https://github.com/liyuming1978/caffe_example/blob/master/install_scripts/Windows_install/build-clcaffe.cmd

test-result:
**ubuntu16.04**:
72.7(fp32)                         72.1(fp16)
**windows**                         
33.49(fp32)                                          10.14(fp16)

**Comment by liyuming1978** [comment]
I compare all layers data,  the reason is dw conv is different with linux and windows

**Comment by liyuming1978** [comment]
"-cl-fast-relaxed-math " in create_dw_conv_kernel
63.55(fp32) 10.14(fp16)

**Comment by liyuming1978** [comment]
delete Fix a potential compiling error in dwconv kernel.  
"+"if (channel_id > 0xfffffffeul) {",    // NOLINT
 +"convolved_image[0] = local_kernel_data[reg];",    // NOLINT
 +"}",    // NOLINT"

63.55(fp32) 54.67(fp16)

**Comment by gongzg** [comment]
This looks like a Windows OCL driver issue.

**Comment by liyuming1978** [comment]
yes, I think so, still has gap  (72.2 -- 63.55)  , in investigation.

**Comment by liyuming1978** [comment]
give up...

**Comment by liyuming1978** [comment]
i find the same data in Ubuntu with 5.0 ocl driver.
